.github/ISSUE_TEMPLATE.md

```diff
@@ -10,10 +10,14 @@
 `crash.log` is a file that is automatically created when KOReader crashes. It can
 normally be found in the KOReader directory:
 
+* `/mnt/private/koreader` for Cervantes
 * `koreader/` directory for Kindle
 * `.adds/koreader/` directory for Kobo
 * `applications/koreader/` directory for Pocketbook
 
+In the case of Android there's no log file because Google restricts what apps can log, so you'll need to obtain logs using `adb logcat KOReader:I ActivityManager:* AndroidRuntime:* *:F`
```

Comment:
```suggestion
Android won't have a crash.log file because Google restricts what apps can log, so you'll need to obtain logs using `adb logcat KOReader:I ActivityManager:* AndroidRuntime:* *:F`.
```